modules/Microsoft.DocumentDB/databaseAccounts/sqlDatabases/deploy.bicep

```diff
@@ -10,6 +10,9 @@ param containers array = []
 @description('Optional. Request units per second.')
 param throughput int = 400
 
+@description('Optional. Specifies the Autoscale settings and represents maximum throughput, the resource can scale up to. If value is set to 0, then the property will be set to null.')
+param autoscaleSettingsMaxThroughput int = 0
```

Comment:
I wonder if 0 isn't a valid input to, in theory, disable the throughput. In other cases we've used '-1' as the switch as this value doesn't make any sense to set otherwise

Reply:
Good call I have aligned to our implementation and provided additional guidance in the parameter